app/views/insured/families/_marriage_fields.html.erb

```diff
@@ -1,24 +1,49 @@
-<div class="special_qle_reasons hidden">
-  <h4> Did you or your spouse have health insurance for at least one day between <%= @qle_date_calc %> and <%= @qle_date %> ? </h4> <br/>
-  <div class = 'moving_radio_btn'>
-    <div class="n-radio-group">
-      <div tabindex="0" onkeydown="handleButtonKeyDown(event, 'qle_submit_reason')" onfocus="handleSEPRadioButton('reason_accept1')" class="n-radio-row">
-        <label class="n-radio vertically-aligned-row" for="reason_accept1">
-          <%= radio_button_tag :reason, "no", false, id: 'reason_accept1', class: "n-radio" %>
-          <span class = "n-radio"></span>
-            yes
-        </label>
-      </div>
-      <div tabindex="0" onkeydown="handleButtonKeyDown(event, 'qle_submit_reason')" onfocus="handleSEPRadioButton('reason_accept')" class="n-radio-row">
-        <label class="n-radio" for="reason_accept">
-        <%= radio_button_tag :reason, "yes", true, id: 'reason_accept', class: "n-radio" %>
-        <span class="n-radio"></span>
-          no
-        </label>
+<% if @bs4 %>
+  <div class="special_qle_reasons hidden">
```

Comment:
supppper nitpick: css class naming best practice is kebob case - no worries about changing this one bc it might be tied to JS or something, just for your future reference

Reply:
yeah this was a pre-existing class used in `qle.js`. A bunch of the classes there use snake case... I did have to add some classes on this PR and mine do use kebob